### ISSUE TITLE
Backup published docs from Gustav

### ISSUE THREAD

**Issue by ennru** [akka/akka.github.com#652]
### Short description

Make sure we can re-create the doc.akka.io sites in case Gustav leaves us for some reason.

### Details

AFAIK, we don't have a copy of the docs published on Gustav more than the local Git repository.
An easy solution would be to push the Git repo to an either public or private Github repo.
We might consider dropping the current one and recreating it with the current state to make it smaller.

**Comment by patriknw** [comment]
Yes please. If size of github repo is a problem or difficulties with credentials we could also copy to moxie servers.

**Comment by ennru** [comment]
The current docs are now pushed to https://github.com/akka/docs.akka.io

The update is not automated, yet.
